tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1754,11 +1754,27 @@ def window(self, size, shift=None, stride=1, drop_remainder=False):
 
     A "window" is a finite dataset of flat elements of size `size` (or possibly
     fewer if there are not enough input elements to fill the window and
-    `drop_remainder` evaluates to false).
+    `drop_remainder` evaluates to `False`).
+
+    The `shift` argument determines the number of input elements by which
+    the window moves on each iteration.  The first element in the `k`th window
+    will be element
+    ```
```

Comment:
put newlines around the "```" to avoid markdown formatting issues

Reply:
Added newlines here in commit 84bc9b4. Also modified the `.pbtxt` file.